packages/venia-concept/src/components/CmsBlock/cmsBlock.js

```diff
@@ -0,0 +1,57 @@
+import { Component, createElement } from 'react';
+import { PropTypes } from 'prop-types';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import classify from 'src/classify';
+import defaultClasses from './cmsBlock.css';
+
+const cmsBlockQuery = gql`
+    query cmsBlocks($identifiers: [String]!) {
+        cmsBlocks(identifiers: $identifiers) {
+            items {
+                content
+            }
+        }
+    }
+`;
+
+class CmsBlock extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            root: PropTypes.string,
+            content: PropTypes.string
+        }),
+        identifiers: PropTypes.oneOfType([PropTypes.string, PropTypes.array])
```

Comment:
Is it normal in Magento frontend to concatenate several CMS blocks together like this? Or are you allowing arrays here just because the GraphQL schema specifies a collection?

If it's not normal to concatenate blocks like this, then I think it would be best to treat this as a scalar "single-block" component, and show an error if multiple results are returned (which should never occur if you're using identifiers).

Reply:
There is such behaviour in graphQl schema. )
That is why did it in a same way.

So, 100% agree with you, in case of simplicity let`s change it.